options/options.go

```diff
@@ -88,7 +89,7 @@ const (
 )
 
 // TerragruntOptions represents options that configure the behavior of the Terragrunt program
-type TerragruntOptions struct {
+type TerragruntOptions struct { //nolint: govet
```

Comment:
Why are you doing this? Just put `Telemetry` with other pointers for the sake of alignment.


You can use `golangci-lint run --fix` to put it in the right place automatically, but it will require that you redo the comments.

Reply:
Thanks. Fixed.